resources/app/lr-aggregator.yaml

```diff
@@ -0,0 +1,124 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: lr-aggregator
+  namespace: kube-system
+  labels:
+    app: lr-aggregator
+spec:
+  type: ClusterIP
+  ports:
+    - name: rpc
+      protocol: TCP
+      port: 9966
+  selector:
+    app: lr-aggregator
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: lr-aggregator
+  namespace: kube-system
+  labels:
+    app: lr-aggregator
+data:
+  logrange.json: |
+    {
+      "HostHostId": 0,
+      "HostLeaseTTLSec": 5,
+      "HostRegisterTimeoutSec": 0,
+      "PublicApiRpc": {
+        "ListenAddr": "0.0.0.0:9966",
+        "TlsEnabled": false,
+        "Tls2Way": false,
+        "TlsCertFile": "/opt/logrange/config/tls/server.crt",
+        "TlsKeyFile": "/opt/logrange/config/tls/server.key",
+        "TlsCAFile": "/opt/logrange/config/tls/ca.pem"
+      },
+      "JrnlCtrlConfig": {
```

Comment:
I really advise against abbreviations unless they're well known or there's a limit on number of characters in the attribute name.

Reply:
took into account, not yet changed